site/content/overview/gateway-api-compatibility.md

```diff
@@ -43,6 +43,18 @@ Each resource below includes the support status of their corresponding fields.
 
 For a description of each field, visit the [Gateway API documentation](https://gateway-api.sigs.k8s.io/references/spec/).
 
+### GatewayClass
```

Comment:
nit: can we also move GatewayClass to be the top element in the summary table at the top?